pkg/bitcoin/electrum/electrum.go

```diff
@@ -519,6 +519,40 @@ func (c *Connection) getConfirmedScriptHistory(
 	return confirmedItems, nil
 }
 
```

Comment:
Could you please add some brief description of this function?

Reply:
Please see cb79db16